platform/core/src/classes/metadata/StudyMetadata.js

```diff
@@ -653,9 +683,13 @@ function _getDisplaySetFromSopClassModule(
  * @param {*} a - DisplaySet
  * @param {*} b - DisplaySet
  */
-function _sortBySeriesNumber(a, b) {
+function sortBySeriesNumber(a, b) {
```

Comment:
I believe we used an underscore to indicate that the method lived in this module and was not exported.

Reply:
Makes sense but since things are not exported by default it's unnecessary. Please let me know if I should return to what it was (in which case I'll have to adopt the leading underscore to the new local function I created such that all non-exported functions start with that instead of just that one)